crypto/digestible/derive/src/lib.rs

```diff
@@ -545,6 +547,112 @@ fn try_digestible_enum(
     Ok(expanded.into())
 }
 
+fn try_digestible_enum_transparent(
+    ident: &Ident,
+    generics: &Generics,
+    variant_data: &DataEnum,
+) -> Result<TokenStream, &'static str> {
+    let (call, call_allow_omit) : (Vec<proc_macro2::TokenStream>, Vec<proc_macro2::TokenStream>) = variant_data
+        .variants
+        .iter()
+        .map(|variant| {
+            let variant_ident = &variant.ident;
+
+            // Our behavior differs based on whether the enum variant is a unit (has no data
+            // associated with it), or named (has struct data associated with it), or unnamed (has
+            // tuple data associated with it).
+            //
+            // We can only support unit variants and unnamed tuples of size 1, in which case we skip right to digesting that.
```

Comment:
not sure if there is a good comment to put here, but the "only support unit variants and ..." is now wrong.
```suggestion
```